src/main/kotlin/org/jmailen/gradle/kotlinter/KotlinterPlugin.kt

```diff
@@ -16,6 +18,15 @@ import java.io.File
 
 class KotlinterPlugin : Plugin<Project> {
 
+    private object Versions {
+        const val ktlint = "0.45.2"
```

Comment:
Ideally this should be somehow shared with the build script.

Reply:
Would it prevent customizing the ktlint version?

Reply:
Today users can select a different ktlint dependency, overriding the one we depend on by default. Would they be able to use the new `ktlint` configuration to do the same? It seems we hardcode it in this implementation.